bugbug/bug_features.py

```diff
@@ -230,6 +230,11 @@ def __call__(self, bug, **kwargs):
         return sum(res) / len(res)
 
 
+class commit_no_of_backouts(object):
+    def __call__(self, bug, **kwargs):
+        return sum(1 for commit in bug['commits'] if commit['ever_backedout'] is True)
```

Comment:
Forgot to tell you before, but here you don't need `is True`

Reply:
`return sum(1 for commit in bug['commits'] if commit['ever_backedout'])`

Reply:
Ohh thanks!Just removed it.